src/nrnpython/grids.cpp

```diff
@@ -100,7 +100,7 @@ ECS_Grid_node *ECS_make_Grid(PyHocObject* my_states, int my_num_states_x,
         new_Grid->proc_offsets = (int*)malloc(nrnmpi_numprocs*sizeof(int));
         new_Grid->proc_num_currents = (int*)malloc(nrnmpi_numprocs*sizeof(int));
```

Comment:
It's probably a good idea to initialize them here too. But it has not caused a problem because currents are initialized from python (in species.py), because currents are common to most models. Node fluxes weren't initialized until the modeler calls node.include_flux.